### PR TITLE
Add studio animation

### PR DESCRIPTION
**Pull request by chris-kuehn**

### CODE REVIEW

File: components/Navigation.js [hunk 1]
```diff
@@ -26,8 +26,11 @@ export default function Navigation({ pageStatus }) {
 const StyledGlobalNavigation = styled.nav`
   position: fixed;
   top: 0;
-  width: 375px;
+  width: 100%;
+  min-width: 375px;
+  max-width: 600px;
   background-color: var(--app-black);
+  z-index: 10;
```
Comment (MerleMak) [inline components/Navigation.js:33]:
```suggestion
  z-index: 1;
```
usually one is enough :) maybe 2 if you use it somewhere else

Comment (MerleMak) [reply components/Navigation.js:33]: please make sure to delete commented out code (like below↓) before pushing :)

File: components/StudioHeader.js [hunk 2]
```diff
@@ -1,32 +1,197 @@
...
+      </StyledContentContainer>
     </StyledStudioHeader>
   );
 }
 
 const StyledStudioHeader = styled.header`
-  background-color: var(--primary-color);
-  width: 375px;
-  height: 375px;
+  /* background-color: var(--primary-color); */
```
Comment (MerleMak) [inline components/StudioHeader.js:161]:
```suggestion
```

File: components/StudioHeader.js [hunk 3]
```diff
@@ -1,32 +1,197 @@
...
+  width: 100%;
+  min-height: 375px;
+  max-height: 600px;
+  height: 100vw;
   margin: 0 auto;
   margin-top: 45px;
+  /* padding: 32px; */
+  /* display: flex;
+  flex-direction: column;
+  justify-content: end; */
```
Comment (MerleMak) [inline components/StudioHeader.js:173]:
```suggestion
```

File: components/StudioHeader.js [hunk 4]
```diff
@@ -1,32 +1,197 @@
...
 `;
 
-const StyledStudioHeadline = styled.h1`
-  padding-bottom: 0.5rem;
+const BackgroundAnimation = styled.canvas`
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  z-index: -1;
+  /* background-color: yellow; */
```
Comment (MerleMak) [inline components/StudioHeader.js:196]:
```suggestion
```

**Review by MerleMak — COMMENTED** [review]
I loooove the way our app looks! Really nice animation! 
Congrats on this nice Capstone project 😍 I only left some comments on the form (commented out code etc.) 
I also clicked through the App and two features came to mind that I would love to see at some point (but no worries, do them in your own time if you want to do them, no pressure)

- Being able to name your Artworks (but maybe you will need a form for that after all :D)
- Being able to control the background color of the artwork

These are great tasks that you can tackle after the bootcamp to stay in the loop to not forget how to code :P

File: components/Navigation.js (hunk 1 above)
Comment (chris-kuehn) [reply components/Navigation.js:33]: Ichanged it to one and it works perfectly :)

File: components/StudioHeader.js (hunk 2 above)
Comment (chris-kuehn) [reply components/StudioHeader.js:161]: deleted! :)

File: components/StudioHeader.js (hunk 3 above)
Comment (chris-kuehn) [reply components/StudioHeader.js:173]: Deleted

File: components/StudioHeader.js (hunk 4 above)
Comment (chris-kuehn) [reply components/StudioHeader.js:196]: deleted!

**Comment by chris-kuehn** [comment]
Thanks for your nice feedback! :) I am happy you like the Look :)

The ideas for the two task i will keep in mind and work on it! :)

**Review by MerleMak — APPROVED** [review]
Yeeeeey! Nice job!

**Pull request merged**